low-level/files/opt/seagate/sspl/setup/resource_map/lr2/server.py

```diff
@@ -285,3 +309,142 @@ def get_fans_info(self):
 
             data.append(fan_dict)
         return data
+
+    def initialize_systemd(self):
+        """Initialize system bus and interface."""
+        self._bus = SystemBus()
+        self._systemd = self._bus.get_object(
+            SYSTEMD_BUS, "/org/freedesktop/systemd1")
+        self._manager = Interface(self._systemd, dbus_interface=MANAGER_IFACE)
+
+    @staticmethod
+    def initialize_cortx_build_info():
+        """Initialize Cortx build info."""
+        if os.path.isfile(CORTX_RELEASE_FACTORY_INFO):
+            cortx_build_info_file_path = "%s://%s" % (
+                CONFIG_SPEC_TYPE, CORTX_RELEASE_FACTORY_INFO)
+            Conf.load("cortx_build_info", cortx_build_info_file_path)
+
+    @staticmethod
+    def get_cortx_service_info():
+        """Get cortx service info in required format."""
+        service_info = []
+        # TODO Get list of services from HA
```

Comment:
Populate this with static list as most basic impl
And keep TODO to get these service names dynamically, the way you showed that day to get PCS resource names from PCS and then getting respective software service names

Reply:
Added static list of Cortx services and TODO comment.